.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Found a bug? Create a report to help us improve!
```

Comment:
I think "and help us" would be good here, to imply we want them involved in the process beyond just submitting reports.